util/__init__.py

```diff
@@ -44,3 +45,5 @@ def utc_now():
     return datetime.datetime.utcnow().isoformat()
 
 
+def monkey_patch_hca_config():
```

Comment:
This is an "interesting" way to resolve this problem.
What other options did you consider?

Reply:
The ideal solution would be to put in a PR to HCA. This is an interim solution, but it works and it avoids messing with the internals of the config file. The problem was really old settings from HCA configs being used and then amended. If they were erased before, I imagine there would have been no config issues with the code. Doing the monkey patch allows us to just change the name of the config and avoid trying to manipulate the same set of configurations for so totally separate purposes.

Reply:
Sounds good, I submitted this issue to HCA:
Problems accessing an alternate DSS from user scripts or unit tests #170
https://github.com/HumanCellAtlas/dcp-cli/issues/170

I have also changed calls to HCAConfig init to include parameters to not save the settings, to help prevent extraneous files from being created: `save_on_exit=True, autosave=False`